### PR TITLE
Fix bad builds by documenting polymer behaviors

### PR DESCRIPTION
**Pull request by FredKSchott**
@abdonrd we just got our new analyzer working in polymer-build, so I decided to take it for a spin. Error reporting still isn't where we want it to be, but it looks like your builds weren't completing because the analyzer was having trouble detecting your behaviors. See https://www.polymer-project.org/1.0/docs/tools/documentation#behaviors for information on how to explicitly document them.

Polymer serve is now working on both bundled and unbundled on my machine. Let me know if you're still seeing problems

### CODE REVIEW

**Comment by abdonrd** [comment]
Thank @FredKSchott! 👏

But I just checked and the problem persists. 😕
With your PR, In Safari, the both (`bundled` and `unbundled`) versions:

screen shot 2016-09-24 at 02 31 36

**Comment by JGSolutions** [comment]
Can this a similar issue:
https://github.com/PolymerElements/app-storage/issues/67#issuecomment-249496650

File: src/app-shared-elements/gdgspain-animate-page.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
 <link rel="import" href="../../bower_components/neon-animation/neon-animation-runner-behavior.html">
 
 <script>
+  /**
+   * @polymerBehavior Polymer.GDGSpainAnimatePageImpl
```
Comment (abdonrd) [inline src/app-shared-elements/gdgspain-animate-page.html]: This should not be without `Impl`?

`* @polymerBehavior Polymer.GDGSpainAnimatePage`

File: src/app-shared-elements/gdgspain-animate-page.html [hunk 2]
```diff
@@ -62,6 +65,9 @@
     }
   };
 
+  /**
+   * @polymerBehavior Polymer.GDGSpainAnimatePage
```
Comment (abdonrd) [inline src/app-shared-elements/gdgspain-animate-page.html]: This should not be without name? Only:

`* @polymerBehavior`

File: src/app-shared-elements/gdgspain-cascaded-animation.html [hunk 3]
```diff
@@ -12,6 +12,10 @@
 <link rel="import" href="../../bower_components/neon-animation/neon-animation-runner-behavior.html">
 
 <script>
+
```
Comment (abdonrd) [inline src/app-shared-elements/gdgspain-cascaded-animation.html]: Please, remove this extra space. 🙈

**Review by abdonrd — CHANGES_REQUESTED** [review]

**Pull request merged**